# low-levelCallExperiment
what happens to a failing low-level call which send eth to a contract without fallback|receive fn?

Below image is the result from calling a function which sends eth to different addresses.
Here I send 2 eth in the fn call.

![experimentResult](./low_levelCAllExperiment.png?raw=true "ExperimentResult")
